source/new.rst

```diff
@@ -536,7 +536,17 @@ b. Bob receives the encrypted "vg-member-added" message
    Bob's device shows
    "You successfully joined the verified group ``GROUP``".
 
-c. Alice's device receives the "vg-member-added-received" reply from Bob
+c. Any other group member that receives the encrypted "vg-member-added" message
+   will process the gossiped key through autocrypt gossip mechanisms.
+   In addition they
+   * Check that the encryption and Alices signature are intact.
+   * Confirm that they are themselves a member of ``GROUP``.
+   * Confirm that Alice is a member of ``GROUP``.
```

Comment:
i'd use "verify" or "check" consistently. "confirm" almost sounds like something is sent back.

Reply:
otherwise +1 for merging.